client.go

```diff
@@ -443,6 +444,10 @@ func (c *Client) DiffService() diff.DiffService {
 	return diffservice.NewDiffServiceFromClient(diffapi.NewDiffClient(c.conn))
 }
 
+func (c *Client) PluginService() introspectionapi.IntrospectionClient {
```

Comment:
should this method name be introspection something?

Reply:
Fixed.